src/pages/home/sidebar/SidebarLinks.tsx

```diff
@@ -22,43 +22,40 @@ import KeyboardShortcut from '@libs/KeyboardShortcut';
 import Navigation from '@libs/Navigation/Navigation';
 import onyxSubscribe from '@libs/onyxSubscribe';
 import Performance from '@libs/Performance';
+import type {OptionData} from '@libs/ReportUtils';
 import SidebarUtils from '@libs/SidebarUtils';
 import * as ReportActionContextMenu from '@pages/home/report/ContextMenu/ReportActionContextMenu';
-import safeAreaInsetPropTypes from '@pages/safeAreaInsetPropTypes';
 import variables from '@styles/variables';
 import * as App from '@userActions/App';
 import * as Session from '@userActions/Session';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type {Modal} from '@src/types/onyx';
 import SignInOrAvatarWithOptionalStatus from './SignInOrAvatarWithOptionalStatus';
 
-const basePropTypes = {
-    /** Toggles the navigation menu open and closed */
-    onLinkClick: PropTypes.func.isRequired,
+type SidebarLinksProps = {
+    onLinkClick: () => void;
 
     /** Safe area insets required for mobile devices margins */
-    insets: safeAreaInsetPropTypes.isRequired,
-};
+    insets?: EdgeInsets;
 
-const propTypes = {
-    ...basePropTypes,
+    isCreateMenuOpen?: boolean;
 
-    optionListItems: PropTypes.arrayOf(PropTypes.string).isRequired,
+    optionListItems: string[] | null;
 
-    isLoading: PropTypes.bool.isRequired,
+    isLoading: OnyxEntry<boolean>;
 
-    // eslint-disable-next-line react/require-default-props
-    priorityMode: PropTypes.oneOf(_.values(CONST.PRIORITY_MODE)),
+    priorityMode: OnyxEntry<ValueOf<typeof CONST.PRIORITY_MODE>>;
```

Comment:
lets create spereate type for `ValueOf<typeof CONST.PRIORITY_MODE>>` and reuse it in other places in the app